README.md

```diff
@@ -106,30 +102,33 @@ Somewhere in your service that's about to make an outgoing call:
     def before_http_request(request, current_span_extractor):
         op = request.operation
         parent_span = current_span_extractor()
-        if parent_span is None:
-            span = opentracing.tracer.start_trace(operation_name=op)
-        else:
-            span = parent_span.start_child(operation_name=op)
+        outbound_span = opentracing.tracer.start_span(
+            operation_name=op,
+            parent=parent_span
+        )
     
-        span.set_tag('server.http.url', request.full_url)
+        outbound_span.set_tag('server.http.url', request.full_url)
         service_name = request.service_name
         host, port = request.host_port
         if service_name:
-            span.set_tag(tags.PEER_SERVICE, service_name)
+            outbound_span.set_tag(tags.PEER_SERVICE, service_name)
         if host:
-            span.set_tag(tags.PEER_HOST_IPV4, host)
+            outbound_span.set_tag(tags.PEER_HOST_IPV4, host)
         if port:
-            span.set_tag(tags.PEER_PORT, port)
+            outbound_span.set_tag(tags.PEER_PORT, port)
     
-        h_ctx, h_attr = opentracing.tracer.trace_context_to_text(
-            trace_context=span.trace_context)
-        for key, value in h_ctx.iteritems():
+        text_carrier = opentracing.SplitTextCarrier()
+        opentracing.tracer.injector(Format.SPLIT_TEXT).inject(
+            span=outbound_span,
+            carrier=text_carrier
+        )
+        for key, value in text_carrier.tracer_state.iteritems():
             request.add_header(key, value)
```

Comment:
I think adding that prefix is the responsibility of the injector, that's how we implemented it.  The prefix cannot be added by the RPC framework, since there maybe a different framework on the receiving side. But the extractor will be the same as injector here.